packages/flutter/lib/src/cupertino/app.dart

```diff
@@ -471,7 +471,12 @@ class CupertinoScrollBehavior extends ScrollBehavior {
   }
 
   @override
-  Widget buildOverscrollIndicator(BuildContext context, Widget child, ScrollableDetails details) {
+  Widget buildOverscrollIndicator(
```

Comment:
Changing these function signatures is a breaking change. Rather than passing the clipBehavior, can you add it to ScrollableDetails? That is already being passed here. We created ScrollableDetails for this exact reason, so if we needed to supply more information to decorators like scrollbar and indicators, we would not have to break the function signature here. :) 

https://github.com/flutter/flutter/blob/4dec56b2c9326b33b156bb053c8a6733778fedf7/packages/flutter/lib/src/widgets/scrollable.dart#L1310